### PR TITLE
docs(homepage): add customer references section

### LINKED ISSUE #1605: [Support Lambda Powertools]: CPQi (Exadel financial services)
### Organization Name

CPQi (Exadel Financial Services) 

### Your Name

JULIO C CAMPOS NETO

### Your current position

LATAM Chief Architect 

### (Optional) Company logo

https://cpqi.com/wp-content/uploads/MicrosoftTeams-image-59.png

### (Optional) Use case

_No response_

### Also using other Lambda Powertools languages?

- [X] Java
- [ ] TypeScript
- [ ] .NET

### LINKED ISSUE #1613: [Support Lambda Powertools]: IMS
### Organization Name

IMS

### Your Name

Carl Reid

### Your current position

Head of Platform Engineering

### (Optional) Company logo

https://ims.tech/wp-content/uploads/2020/01/cropped-cropped-IMS-Logo.png

### (Optional) Use case

We have used Lamba powertools extensively during the build of DriveSync 6 serverless platform to ensure best practice and improve time to market.

### Also using other Lambda Powertools languages?

- [ ] Java
- [ ] TypeScript
- [ ] .NET

### LINKED ISSUE #1614: [Support Lambda Powertools]: Trek10
### Organization Name

Trek10

### Your Name

Ryan Toler

### Your current position

Cloud Architect

### (Optional) Company logo

https://drive.google.com/file/d/1--XGseX58-su1R2rn8N_4ir1SIxUB1dU/view?usp=sharing

### (Optional) Use case

_No response_

### Also using other Lambda Powertools languages?

- [ ] Java
- [X] TypeScript
- [ ] .NET

### LINKED ISSUE #1644: [Support Lambda Powertools]: CyberArk
### Organization Name

CyberArk

### Your Name

Ran Isenberg

### Your current position

Principal Software Architect

### (Optional) Company logo

https://www.cyberark.com/wp-content/uploads/2021/01/cyberark-logo-dark.svg

### (Optional) Use case

_No response_

### Also using other Lambda Powertools languages?

- [ ] Java
- [ ] TypeScript
- [ ] .NET

### LINKED ISSUE #1648: [Support Lambda Powertools]: CloudZero
### Organization Name

CloudZero

### Your Name

Erik Peterson

### Your current position

Founder and CTO

### (Optional) Company logo

https://downloads.cloudzero.com/assets/cloudzero-logo-4c-with-border.png

### (Optional) Use case

CloudZero is a 100% Serverless company, and we use Lambda Powertools _everywhere_! Logging, tracing, validating input, handling events, you name it, Powertools has become a key part of our stack!

### Also using other Lambda Powertools languages?

- [ ] Java
- [ ] TypeScript
- [ ] .NET

### LINKED ISSUE #1673: [Support Lambda Powertools]: TopSport
### Organization Name

TopSport.com.au

### Your Name

Peter Schutt

### Your current position

Engineer

### (Optional) Company logo

https://www.topsport.com.au/assets/images/logo_pulse.svg

### (Optional) Use case

_No response_

### Also using other Lambda Powertools languages?

- [ ] Java
- [ ] TypeScript
- [ ] .NET

### LINKED ISSUE #1783: [Support Lambda Powertools]: globaldatanet
### Organization Name

globaldatanet

### Your Name

Stephan Huber

### Your current position

AWS Cloud Developer

### (Optional) Company logo

https://raw.githubusercontent.com/globaldatanet/.github/main/profile/logo.png

### (Optional) Use case

_No response_

### Also using other Lambda Powertools languages?

- [ ] Java
- [X] TypeScript
- [ ] .NET

### LINKED ISSUE #1941: [Support Lambda Powertools]: Propellor.ai
### Organization Name

propellor.ai

### Your Name

Dheeraj

### Your current position

Chief Technology Officer

### (Optional) Company logo

https://i0.wp.com/propellor.ai/wp-content/uploads/2022/07/propellor-ai.png?fit=1116%2C261&ssl=1

### (Optional) Use case

We used to use the stock python "Logging" library, but it was hard to figure out the context and traces for the logging library. Added to that structured logging is also as issue.  I'm looking for a version of AWS and found powertools in 2021. We used Powertools to do simple logging till date, but now we want to use it to its fullest potential.

### Also using other Lambda Powertools languages?

- [ ] Java
- [ ] TypeScript
- [ ] .NET

### PR DESCRIPTION
**Pull request by leandrodamascena**
**Issue number:** #1605 
**More related issues:** #1613 #1614 #1648 #1673 #1644 #1783 #1941 

### Changes

Added section on how to support Lambda Powertools.

### User experience

image

## Checklist

If your change doesn't seem to apply, please leave them unchecked.

* [x] [Meet tenets criteria](https://awslabs.github.io/aws-lambda-powertools-python/#tenets)
* [x] I have performed a self-review of this change
* [ ] Changes have been tested
* [ ] Changes are documented
* [x] PR title follows [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-python/blob/develop/.github/semantic.yml)

<details>
<summary>Is this a breaking change?</summary>

**RFC issue number**:

Checklist:

* [ ] Migration process documented
* [ ] Implement warnings (if it can live side by side)

</details>

## Acknowledgment

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Disclaimer**: We value your time and bandwidth. As such, any pull requests created on non-triaged issues might not be successful.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,32 @@ With [pip](https://pip.pypa.io/en/latest/index.html) installed, run: ``pip insta
...
 * [Serverless GraphQL Nanny Booking Api](https://github.com/trey-rosius/babysitter_api)
 
+## How to support AWS Lambda Powertools for Python?
+
+### Becoming a reference customer
+
+Knowing which companies are using this library is important to help prioritize the project internally.
+If you would like us to include your company’s name and/or logo in the README file to indicate that your company is using the AWS Lambda Powertools for Python, please raise a [Support Lambda Powertools (become a reference)](https://github.com/awslabs/aws-lambda-powertools-python/issues/new?assignees=&labels=customer-reference&template=support_powertools.yml&title=%5BSupport+Lambda+Powertools%5D%3A+%3Cyour+organization+name%3E) issue.
+
+The following companies, among others, uses Powertools:
```
Comment (heitorlessa) [inline README.md]:
```suggestion
The following companies, among others, use Powertools:
```

**Review by heitorlessa — CHANGES_REQUESTED** [review]
minor request - duplicate it in our user guide docs, please

**Comment by leandrodamascena** [comment]
> minor request - duplicate it in our user guide docs, please

Can you check out @heitorlessa? I kept the banner at the top of the page and added a new section. I think it makes sense because if the user can't see the banner, the customer can see the menu.

**Review by heitorlessa — APPROVED** [review]

**Comment by heitorlessa** [comment]
Done ;) just edited the PR Title so we can easily track in the changelog/search

**Pull request merged**